src/ape/types/__init__.py

```diff
@@ -276,6 +304,25 @@ def get(self, item: str, default: Optional[Any] = None) -> Any:
         return self.event_arguments.get(item, default)
 
 
+class MockContractLog(BaseContractLog):
```

Comment:
- [ ] Add class doc here
- [ ] Include `MockContractLog` in `docs/methoddocs/types.md` members list under the misc header